airqo-platform/AirQo-frontend
# Frontend: Device Incident Reports

- What does this PR do?
This PR finishes the implementation of the Incident Report of the Netmanager's Device Management.
- What JIRA task is related to this PR?
The link to the JIRA board task is [here](https://airqoteam.atlassian.net/browse/NET-6).
- How do I test out this PR?
Git clone this repo `cd AirQo-frontend/locate` Then run `npm install` to get all the necessary dependencies Afterwards run `npm run dev-mac` for MAC users and `npm run dev-pc `for windows users
- Other considerations?
> You should also run the **data-mgt** microservice with the latest branch of incident report (`ft-device-incident-reports`).
> Before that, you will also want to start up the auth service (`AirQo-api/src/auth-service`) so that you can Login accordingly.
